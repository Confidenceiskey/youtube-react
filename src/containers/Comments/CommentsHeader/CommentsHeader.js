import React from 'react';
import { Button, Icon } from 'semantic-ui-react';

const CommentsHeader = ({ amountComments }) => {
  return (
    <div className='comments-header'>
      <h4>{amountComments} Comments</h4>
      <Button basic compact icon labelPosition='left'>
        <Icon name='align left' />
        Sort by
      </Button>
    </div>
  );
}

export default CommentsHeader;
