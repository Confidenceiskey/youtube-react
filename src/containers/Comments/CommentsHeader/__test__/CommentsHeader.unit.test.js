import React from 'react';
import { shallow } from 'enzyme';
import CommentsHeader from '../CommentsHeader';

describe('Comments Header', () => {
  it('CommentsHeader renders w/ amountComments=null', () => {
    const wrapper = shallow(
      <CommentsHeader />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('CommentsHeader renders w/ amountComments = number ', () => {
    const wrapper = shallow(
      <CommentsHeader amountComments={123} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});