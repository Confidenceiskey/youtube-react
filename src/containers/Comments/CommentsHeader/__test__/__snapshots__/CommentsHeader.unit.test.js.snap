// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments Header CommentsHeader renders w/ amountComments = number  1`] = `
<div
  className="comments-header"
>
  <h4>
    123
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;

exports[`Comments Header CommentsHeader renders w/ amountComments=null 1`] = `
<div
  className="comments-header"
>
  <h4>
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;
